docs/content/guides/3.performance/1.index.md

```diff
@@ -0,0 +1,44 @@
+# Introduction to Web Performance
+::subheader
+Improve your website's user experience and increase conversions by optimizing your website's performance.
+::
+
+Web performance is an ever-important subject in modern web. While your Alokai application is built with the best practices in mind, it's important to understand some of the basics of web performance so that as you build out your custom storefront, you can ensure that your users have a great experience.
+
+On the following pages you will find list of good practices that will help you optimize your website performance.
+
+## Monitoring Performance
+
```

Comment:
suggestion: could you add here even a simple sentence explaining what monitoring performance is about? I am not a fan of empty `h2` followed immediately by a huge `h3`